rules/opentitan.bzl

```diff
@@ -825,6 +827,12 @@ def opentitan_binary(
         testonly = testonly,
         **kwargs
     )
+    elf_transition_binary_name = "{}_elf".format(name)
```

Comment:
Now that we have the transition, can we add the "elf_transition_binary_name" target name to the `targets` list? i.e., with a `targets.append(":" + elf_transition_binary_name)` ?

Reply:
Yes absolutely, I missed that the commit. I have a strange failure in CI, I suspect maybe calling the transition target just `<name>_elf` is confusing and error-prone, I will rename it to `<name>_elf_transition` following your suggestion.